### ISSUE TITLE
[fvt]2.12.2 xcatprobe osdeploy -n node -r 24h  could not get right deploy messages

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#1707]
env:all platform
build:
# lsxcatd -v
Version 2.12.2 (git commit f371073d92da1546259d18bf10162636fe2bcaaf, built Mon Aug 15 06:15:55 EDT 2016)
 # 
How to reproduce:
```
on our autotest environment:
[root@c910f02c01p14 ~]# xcatprobe osdeploy -n c910f02c01p08 -r 48h1m
All pre_defined nodes are valid                                                                                   [ OK ]
The installation network interface is eth1                                                                        [INFO]
Start to search logs after 'Sun Aug 14 02:32:46 2016', please waiting for a while.............

=============================================================
=  The summary of os provision:
=============================================================

[c910f02c01p08] deployment failed                                                                                 [FAIL]
        There are more than one possible process satisfy c910f02c01p08 situation                                  
        possible process "deploy", pass unknown stage, something wrong during server_reboot stage                 
        possible process "reboot", pass unknown stage, something wrong during done stage                          
        possible process "reboot", pass unknown stage, something wrong during server_reboot stage                 
        possible process "reboot", pass unknown stage, something wrong during server_reboot stage                 
        possible process "deploy", pass unknown stage, something wrong during loaded_kernel_and_initrd stage      
-------------------> This is out autotest environment and  c910f02c01p08 is our servicenode ,but osdeploy -r  could not get right deploy message
```

**Comment by hu-weihua** [comment]
@junxiawang , I have fixed this issue by pull request #1710 . Could you help to verify it ? thanks

**Comment by junxiawang** [comment]
verified
